infra/values.yaml

```diff
@@ -0,0 +1,15 @@
+# yaml-language-server: $schema=../../infra-helm-charts/charts/discord-bot/values.schema.json
+application:
+  version: "${COMMIT_SHA}"
+  containers:
+    - name: "scuffbot"
+      image: "${DOCKERHUB_USERNAME}/${DOCKERHUB_REPO}:${REPO_NAME}"
+      environment:
+        APPLICATION_ID: /secrets/application-id
+        BOT_TOKEN: /secrets/bot-token
+        CONFIG_FILE: /config/config.yaml
+      config:
+        fileVariable: "scuffbotConfig"
```

Comment:
Here, could we not just have `config: placeholder` and then in the pipeline, substitute it with `--set-file config=config.yaml`?

Reply:
True, we could have it as config: <variable name>. The reason I'm giving the option to code in a variable is for the consideration if multiple containers need each their own config.yaml, if that makes sense